plugins/parcelize/parcelize-compiler/parcelize.common/src/org/jetbrains/kotlin/parcelize/BuiltinParcelableTypes.kt

```diff
@@ -120,4 +120,8 @@ object BuiltinParcelableTypes {
         "kotlin.collections.Set",
         *IMMUTABLE_COLLECTIONS_FQNAMES.toTypedArray(),
     )
+
+    val CUSTOM_SUPPORTED_FQNAMES = setOf(
```

Comment:
I wonder if we want a separate set for this? This is  like `ULong` which we have in the `PARCELABLE_BASE_TYPE_FQNAMES` set. Should we just fold this in with that and add the case for it to the big switch over all those? Since this is like `ULong` it is not clear what the distinction is with `CUSTOM_SUPPORTED_FQNAMES`.

Reply:
Done